mod/da/pkg/store/store.go

```
@@ -22,15 +22,18 @@
 
 	"github.com/berachain/beacon-kit/mod/da/pkg/types"
-	"github.com/berachain/beacon-kit/mod/errors"
 	"github.com/berachain/beacon-kit/mod/log"
 	"github.com/berachain/beacon-kit/mod/primitives/pkg/common"
 	"github.com/berachain/beacon-kit/mod/primitives/pkg/math"
-	"github.com/sourcegraph/conc/iter"
 )
 
+// SLOT_COMMITMENTS_KEY is the key used to store the commitments for a slot
+// in the DB. We use this key to avoid conflicts with the slot index.
+const SLOT_COMMITMENTS_KEY = "slot_commitments"
```

golangci-lint

[warning] var-naming: don't use ALL_CAPS in Go names; use CamelCase

(revive)